confluence/api.go

```diff
@@ -113,15 +139,25 @@ func NewAPI(baseURL string, username string, password string) *API {
 		rest.SetHeader("Authorization", fmt.Sprintf("Bearer %s", password))
 	}
 
+	restV2 := gopencils.Api(baseURL+"/api/v2", auth, 3) // v2 API for folders and new features
```

Comment:
Is there a minimum confluence version that is required for this api?

Reply:
From what I can tell, the v2 API is the new default, and v1 has been deprecated: https://community.developer.atlassian.com/t/rfc-19-deprecation-of-confluence-cloud-rest-api-v1-endpoints/71752